pkg/net/obd.go

```diff
@@ -104,12 +104,12 @@ func ComputeOBDStats(latencies, throughputs []float64) (Latency, Throughput, err
 		return Latency{}, Throughput{}, err
 	}
 	t := Throughput{
-		Avg:          avgThroughput,
-		Percentile50: percentile50Throughput,
-		Percentile90: percentile90Throughput,
-		Percentile99: percentile99Throughput,
-		Min:          minThroughput,
-		Max:          maxThroughput,
+		Avg:          avgThroughput * 8,
```

Comment:
why are we changing this now? @wlan0 - can you explain what was the issue? as this will make our current `mc` release incompatible?

Reply:
discussed this offline. This change is not needed